libraries/AP_HAL_ChibiOS/hwdef/AEROFOX-H7/defaults.parm

```diff
@@ -0,0 +1,6 @@
+ARSPD_TYPE	8
+BATT_VOLT_PIN	19
+BATT_VOLT_MULT	21
+BATT_MONITOR	3
+CAN_P1_DRIVER	1
+Q_ENABLE 1

```

Comment:
unless this is going to only be used in QuadPlanes as a turn key system, remove this line